lua/telescope/builtin/init.lua

```diff
@@ -21,60 +30,276 @@ end
+---@field show_line boolean: if true, shows the row:column that the result is found at (default is true)
 builtin.treesitter = require('telescope.builtin.files').treesitter
+
+--- Live fuzzy search inside of the currently open buffer
+---@param opts table: options to pass to the picker
 builtin.current_buffer_fuzzy_find = require('telescope.builtin.files').current_buffer_fuzzy_find
+
+--- Lists tags in current directory with tag location file preview (users are required to run ctags -R to generate tags
+--- or update when introducing new changes)
+---@param opts table: options to pass to the picker
+---@field ctags_file string: specify a particular ctags file to use
+---@field show_line boolean: if true, shows the content of the line the tag is found on in the picker (default is true)
+---@field shorten_path boolean: if true, makes file paths shown in picker use one letter for folders (default is true)
+---@field hide_filename boolean: if true, hides the name of the file in the current picker (default is false)
 builtin.tags = require('telescope.builtin.files').tags
+
+--- Lists all of the tags for the currently open buffer, with a preview
+---@param opts table: options to pass to the picker
 builtin.current_buffer_tags = require('telescope.builtin.files').current_buffer_tags
 
+
+--
+--
+-- Git-related Pickers
+--
+--
+--- Fuzzy search through the output of `git ls-files` command, respects .gitignore, optionally ignores untracked files
+---@param opts table: options to pass to the picker
+---@field show_untracked boolean: if true, adds the `--others` flag to the search (default is true)
+---@field recurse_submodules boolean: if true, adds the `--recurse-submodules` flag to the search (default is false)
 builtin.git_files = require('telescope.builtin.git').files
+
+--- Lists commits for current directory, with diff preview and checkout on <cr>
+---@param opts table: options to pass to the picker
 builtin.git_commits = require('telescope.builtin.git').commits
+
+--- Lists commits for current buffer, with diff preview and checkout on <cr>
+---@param opts table: options to pass to the picker
 builtin.git_bcommits = require('telescope.builtin.git').bcommits
+
+--- List branches for current directory, with log preview
+--- - Default keymaps:
+---   - <C-t>: tracks currently selected branch
+---   - <C-r>: rebases currently selected branch
+---   - <C-a>: creates a new branch, with confirmation prompt before creation
+---   - <C-d>: deletes the currently selected branch, with confirmation prompt before deletion
+---@param opts table: options to pass to the picker
 builtin.git_branches = require('telescope.builtin.git').branches
+
+--- Lists git status for current directory
+--- - Default keymaps:
+---   - <Tab>: stages or unstages the currently selected file
+---@param opts table: options to pass to the picker
 builtin.git_status = require('telescope.builtin.git').status
 builtin.git_stash = require('telescope.builtin.git').stash
```

Comment:
`git_stash` seems to be missing

Reply:
OOPS I forgot that was added recently. Good catch lol